application/org.openjdk.jmc.flightrecorder.ui/src/main/resources/org/openjdk/jmc/flightrecorder/ui/messages/internal/messages.properties

```diff
@@ -271,6 +271,7 @@ FlavorSelector_BUTTON_TIMERANGE_CLEAR_TOOLTIP=Clear time range zoom on page
 FlavorSelector_BUTTON_TIMERANGE_SET=Set
 FlavorSelector_BUTTON_TIMERANGE_SET_TOOLTIP=Zoom to time range on relevant places on the page
 FlavorSelector_LABEL_ASPECT=Aspect:
+FlavorSelector_LABEL_SELECTION=Selection:
```

Comment:
Selection is a bit of an overloaded term. Perhaps Focus would be better (it also takes less screen estate)? Check with a few more people to see which makes the more sense. 

The action is "Set as As Focused Selection".